Aquality.Selenium/src/Aquality.Selenium/Configurations/WebDriverSettings/EdgeSettings.cs

```diff
@@ -21,18 +20,33 @@ public EdgeSettings(ISettingsFile settingsFile) : base(settingsFile)
 
         protected override BrowserName BrowserName => BrowserName.Edge;
 
+        public override string DownloadDirCapabilityKey => "download.default_directory";
+
         public override DriverOptions DriverOptions
         {
             get
             {
                 var options = new EdgeOptions();
-                SetCapabilities(options);
-                SetOptionsByPropertyNames(options);
+                SetEdgeChromiumPrefs(options);
+                SetCapabilities(options, (name, value) => options.AddAdditionalOption(name, value));
+                SetEdgeChromiumArguments(options);
```

Comment:
if we removed the `EdgeChromium`, then maybe we should remove "`Chromium`" from the names of this methods?
like `SetEdgePrefs`

Reply:
fixed